mix.exs

```diff
@@ -12,7 +12,6 @@ defmodule ImgProxy.MixProject do
       elixir: "~> 1.8",
       start_permanent: Mix.env() == :prod,
       deps: deps(),
-      description: "imgproxy URL generator and helper functions",
```

Comment:
Was this removal intentional?

Reply:
It's moved under `package/0`, which have the same effect, but more relevant.